vscode-visual-tests/features/analytics_extension_check_on_cli.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Visual Studio Code + the Analytics plugin basic functionality
+
+    Scenario: Check that the Analytics plugin can be found in the Marketplace
+        Given The PyAutoGUI library is initialized
+          And The screen resolution is at least 1024x768 pixels
```

Comment:
Do we need this minimum resolution? And it is at many places. Is there any such limit by VsCode?

Reply:
Yes, it is to ensure that the layout will stay the same. On smaller screen (but who have less than 1024x768 nowadays?) the layout was not consistent.